pom.xml

```diff
@@ -99,6 +99,11 @@
 			<artifactId>aws-java-sdk-ec2</artifactId>
 			<version>${amazon.sdk.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>com.amazonaws</groupId>
```

Comment:
Could we instead use the new sdk? (<groupId>software.amazon.awssdk</groupId>)

Reply:
Done!